index.js

```diff
@@ -52,4 +52,145 @@ const state = {
     }
   ],
   cart: []
-};
\ No newline at end of file
+};
+  
+  // //select the ul element
+  // const ul = document.querySelector(".item-list")
+    
+  //   //create li elements in the ul for each item
+  //   let li = document.createElement("li")
+  //   console.log(li)
+    
+  //   const imgDiv = document.createElement("div")
+  //   imgDiv.className = "store--item-icon"
+  //   li.append(imgDiv)
+    
+    
+  //   const img = document.createElement("img")
+  //   img.src = "assets/icons/001-beetroot.svg"
+  //   imgDiv.append(img)
+    
+  //   const button = document.createElement("button")
+  //   button.innerHTML = "Add to Cart!"
+  //   li.append(button)
+    
+  //   ul.append(li)
+    
+
+// array of images
+    
+  const images = [
+    {
+      src: "assets/icons/001-beetroot.svg",
+      buttonText: "Add to Cart!"
+    },
+    {
+      src: "assets/icons/002-carrot.svg",
+      buttonText: "Add to Cart!"
+    },
+    {
+      src: "assets/icons/003-apple.svg",
+      buttonText: "Add to Cart!"
+    },
+    {
+      src:"assets/icons/004-apricot.svg",
+      buttonText: "Add to Cart!",
+    },
+    {
+      src:"assets/icons/005-avocado.svg",
+      buttonText: "Add to Cart!",
+    },
+    {
+      src:"assets/icons/006-bananas.svg",
+      buttonText: "Add to Cart!",
+    },
+    {
+      src:"assets/icons/007-bell-pepper.svg",
+      buttonText: "Add to Cart!",
+    },
+    {
+      src:"assets/icons/008-berry.svg",
+      buttonText: "Add to Cart!",
+    },
+    {
+      src:"assets/icons/009-blueberry.svg",
+      buttonText: "Add to Cart!",
+    },
+    {
+      src:"assets/icons/010-eggplant.svg",
+      buttonText: "Add to Cart!",
+    }
+  ];
+  
```

Comment:
thats clever, to create an array with the images and buttons ahead of the function